charts/astronomer/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: astronomer
-version: 0.12.0
+version: 0.13.0
```

Comment:
we discussed that these don't matter very much, so if there was a merge conflict on one I just bumped to 0.13.0 and we can leave it all the way until the next patch version if we want.